app/views/cdo/create/confirm-all-details.njk

```diff
@@ -58,14 +58,12 @@
                 <a class="govuk-link govuk-link--no-visited-state" href="/cdo/create/owner-details#postcode">Change<span class="govuk-visually-hidden"> address (Owner details)</span></a>
               </dd>
             </div>
-            <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-              <dt class="govuk-summary-list__key">
-                Country
-              </dt>
-              <dd class="govuk-summary-list__value">
-                {{ model.summary.owner.country }}
-              </dd>
-            </div>
+            <dt class="govuk-summary-list__key">
```

Comment:
I assume this is correct based on the prototype - with horizontal border (if relevant)?

Reply:
this is the prototype:
https://dangerous-dogs-index-2616278c1d9a.herokuapp.com/v8/cdo/create/confirm-details
```
<div class="govuk-summary-list__row">
          <dt class="govuk-summary-list__key">
            Address
          </dt>
          <dd class="govuk-summary-list__value">
            12 Primrose Road<br>Southampton<br>SO16 3GP
          </dd>

            <dd class="govuk-summary-list__actions">

                <a class="govuk-link govuk-link--no-visited-state" href="#">Change<span class="govuk-visually-hidden"> address (Owner details)</span></a>
            </dd>
        </div>

          <dt class="govuk-summary-list__key govuk-!-padding-top-0">
            Country
          </dt>
          <dd class="govuk-summary-list__value govuk-!-padding-top-0">
            England
          </dd>```